app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -188,140 +188,158 @@ def disk_mode_types
+        "rhel6_64Guest" => "Red Hat Enterprise Linux 6 (64-bit)",
+        "rhel6Guest" => "Red Hat Enterprise Linux 6 (32-bit)",
+        "rhel7_64Guest" => "Red Hat Enterprise Linux 7 (64-bit)",
+        "rhel7Guest" => "Red Hat Enterprise Linux 7 (32-bit)",
+        "sjdsGuest" => "Sun Java Desktop System",
+        "sles10_64Guest" => "Suse Linux Enterprise Server 10 (64-bit)",
+        "sles10Guest" => "Suse Linux Enterprise Server 10 (32-bit)",
+        "sles11_64Guest" => "Suse Linux Enterprise Server 11 (64-bit)",
+        "sles11Guest" => "Suse Linux Enterprise Server 11 (32-bit)",
+        "sles12_64Guest" => "Suse Linux Enterprise Server 12 (64-bit)",
+        "sles12Guest" => "Suse Linux Enterprise Server 12 (32-bit)",
+        "sles64Guest" => "Suse Linux Enterprise Server 9 (64-bit)",
+        "slesGuest" => "Suse Linux Enterprise Server 9 (32-bit)",
+        "solaris10_64Guest" => "Solaris 10 (64-bit)",
+        "solaris10Guest" => "Solaris 10 (32-bit)",
+        "solaris11_64Guest" => "Solaris 11 (64-bit)",
+        "solaris6Guest" => "Solaris 6",
+        "solaris7Guest" => "Solaris 7",
+        "solaris8Guest" => "Solaris 8",
+        "solaris9Guest" => "Solaris 9",
+        "suse64Guest" => "Suse Linux (64-bit)",
+        "suseGuest" => "Suse Linux (32-bit)",
+        "turboLinux64Guest" => "Turbolinux (64-bit)",
+        "turboLinuxGuest" => "Turbolinux (32-bit)",
+        "ubuntu64Guest" => "Ubuntu Linux (64-bit)",
+        "ubuntuGuest" => "Ubuntu Linux (32-bit)",
         "unixWare7Guest" => "SCO UnixWare 7",
-        "darwinGuest" => "Apple Mac OS X 10.5 (32-bit)",
-        "darwin64Guest" => "Apple Mac OS X 10.5 (64-bit)",
-        "darwin10Guest" => "Apple Mac OS X 10.6 (32-bit)",
-        "darwin10_64Guest" => "Apple Mac OS X 10.6 (64-bit)",
-        "darwin11Guest" => "Apple Mac OS X 10.7 (32-bit)",
-        "darwin11_64Guest" => "Apple Mac OS X 10.7 (64-bit)",
-        "darwin12_64Guest" => "Mac OS 10.8 (64-bit)",
-        "darwin13_64Guest" => "Mac OS 10.9 (64-bit)",
-        "vmkernelGuest" => "VMware ESX 4.x",
-        "vmkernel5Guest" => "VMware ESXi 5.x",
-        "vmkernel6Guest" => "VMware ESXi 6.x",
-        "otherGuest" => "Other (32-bit)",
-        "otherGuest64" => "Other (64-bit)"
+        "vmkernel5Guest" => "VMware ESX 5",
+        "vmkernel65Guest" => "VMware ESX 6.5",
+        "vmkernel6Guest" => "VMware ESX 6",
+        "vmkernelGuest" => "VMware ESX 4",
+        "vmwarePhoton64Guest" => "VMware Photon (64-bit)",
+        "win2000AdvServGuest" => "Windows 2000 Advanced Server",
+        "win2000ProGuest" => "Windows 2000 Professional",
+        "win2000ServGuest" => "Windows 2000 Server",
+        "win31Guest" => "Windows 3.1",
+        "win95Guest" => "Windows 95",
+        "win98Guest" => "Windows 98",
+        "windows7_64Guest" => "Windows 7 (64-bit)",
+        "windows7Guest" => "Windows 7 (32-bit)",
+        "windows7Server64Guest" => "Windows Server 2008 R2 (64-bit)",
```

Comment:
Cutting "R2" and "(64-Bit)" ok - but I didnt see sth. compareable in the other MS-Products. Can you specify this that we dont talk at cross-purposes ? Thanks in advance.